### PR TITLE
AST-1088 - E2E Test case for Single post section title font

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added E2E test case for Single post section title font.

### Screenshots

 https://share.getcloudapp.com/Jrunjo0w

### How has this been tested?

Run single test - npm run test:e2e:interactive tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		} ).cssValueToBe( `${ sectionTitleFont[ 'related-posts-section-title-font-weight' ] }` );
+
+		await expect( {
+			selector: '.ast-related-posts-title',
+			property: 'font-size',
+		} ).cssValueToBe( `${ sectionTitleFont[ 'related-posts-section-title-font-size' ].desktop }${ sectionTitleFont[ 'related-posts-section-title-font-size' ][ 'desktop-unit' ] }` );
+
+		await setBrowserViewport( 'medium' );
+		await expect( {
+			selector: '.ast-related-posts-title',
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:60]: @SimranChougule1 test case is failing at local,Please check it

File: tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+				sectionTitleFont[ 'related-posts-section-title-font-size' ].mobile,
+			) }${
+				sectionTitleFont[ 'related-posts-section-title-font-size' ][ 'mobile-unit' ]
+			}`,
+		);
+
+		await expect( {
+			selector: '.ast-related-posts-title',
+			property: 'line-height',
+		} ).cssValueToBe( `${ sectionTitleFont[ 'related-posts-section-title-line-height' ] }` );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:85]: @SimranChougule1 test case failing here,please check it at local.

Comment (GauravKhupse) [reply tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:84]: @SimranChougule1 @bsfsnehal  Line height is multiple of font size in CSS computed, please refer to this article and update:- https://stackoverflow.com/questions/47327375/css-line-height-number-to-px

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js]: @GauravKhupse added code for multiplying line height and font size

File: tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+		} ).cssValueToBe(
+			`${ await responsiveFontSize(
+				sectionTitleFont[ 'related-posts-section-title-font-size' ].mobile,
+			) }${
+				sectionTitleFont[ 'related-posts-section-title-font-size' ][ 'mobile-unit' ]
+			}`,
+		);
+		await expect( {
+			selector: '.ast-related-posts-title',
+			property: 'line-height',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:83]: @bsfsnehal error on font, please check this issue

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:83]: @SimranChougule1 please refer this test case for resolving line-height issue https://github.com/brainstormforce/astra/pull/3344/files

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js (hunk 3 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:87]: Okay

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/blog/single-post/section-title-font.test.js:87]: Yes, as discussed, I have commented responsive font size code and for now line height issue is fixed

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**